cea/optimization/master/generation.py

```diff
@@ -108,7 +107,10 @@ def generate_main(nBuildings, config):
 
         # Allocation of Shares
         def cuts(ind, nPlants, irank):
-            cuts = sorted(random_sample(nPlants - 1) * 0.99 + 0.009)
+            cuts = []
```

Comment:
no. let's keep using numpy for this. check out https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.random.seed.html - then you don't have to change your code. Call this in the main function of the optimization with the config variable.